R/defaults.R

```diff
@@ -86,7 +86,7 @@ epidist.default <- function(data, formula = epidist_formula(data),
     backend = "cmdstanr", data = data, ...
   )
 
-  class(fit) <- c(class(fit), "epidist_fit")
+  class(fit) <- c(class(fit), "epidist_fit", family$name)
```

Comment:
I think if this is the name taken from brms directly which I think it is it could lead to collisions with other packages. Happy for this to be dealt with in an issue.

Reply:
Yeah fair, I'll make an issue. We can just do `epidist_family`